db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.genre_name
```

Comment:
The `__str__` method is trying to return `self.genre_name`, but there is no `genre_name` attribute in the `Genre` model. It should return `self.name` instead, as `name` is the defined attribute.